### PR TITLE
Updated Docker database container with info about setting CRYPT_KEY o…

### PR DESCRIPTION
**Pull request by meker12**
## Purpose of this pull request

- Updated the Docker database container topic with information about setting the encryption key for an imported database using the CRYPT_KEY deploy variable.
- Updated the "Add Magento encryption key" with instructions for setting key using CRYPT_KEY environment variable
- Added release note about new validation that checks the Cloud environment for encryption key when deploying to local Docker environments
- Cleanup markdown syntax

## Affected DevDocs pages

- https://devdocs.magento.com/cloud/docker/docker-database.html
- https://devdocs.magento.com/cloud/env/variables-deploy.html
- https://devdocs.magento.com/cloud/docker/docker-database.html
- https://devdocs.magento.com/cloud/setup/first-time-setup-import-import.html
- https://devdocs.magento.com/cloud/release-notes/mcd-release-notes.html

### CODE REVIEW

File: src/cloud/docker/docker-database.md [hunk 1]
```diff
@@ -33,6 +33,9 @@ To import a database dump into the Docker environment:
 
 Although it is a more complex approach, you can use GZIP by _sharing_ the `.sql.gz` file using the `.docker/mnt` directory and importing it inside the Docker container.
 
+{:.bs-callout-info}
+When you import a database from an existing Magento installation into a new Magento Commerce Cloud environment, you must add the encryption key from the remote environment to the new environment, and then deploy the changes. See [Add the Magento encryption key]({{ site.baseurl}}/cloud/setup/first-time-setup-import-import.html#encryption-key).
```
Comment (hguthrie) [inline src/cloud/docker/docker-database.md]: Use variables for product name.

File: src/cloud/setup/first-time-setup-import-import.md [hunk 2]
```diff
@@ -27,88 +27,87 @@ Create a remote Git reference from your Cloud Git repository to the repository c
...
-         mymagento (type: php:7.0, size: S, disk: 2048)
-         mysql (type: mysql:10.0, size: S, disk: 2048)
-         redis (type: redis:3.0, size: S)
-         solr (type: solr:4.10, size: S, disk: 1024)
-
-    Environment routes:
-       http://master-o9gv6gq-43biovskhelhy.us.magentosite.cloud/ is served by application `mymagento`
-       https://master-o9gv6gq-43biovskhelhy.us.magentosite.cloud/ is served by application `mymagento`
-    ```
+   ```bash
```
Comment (hguthrie) [inline src/cloud/setup/first-time-setup-import-import.md]:
```suggestion
   ```terminal
```

File: src/cloud/setup/first-time-setup-import-import.md [hunk 3]
```diff
@@ -124,88 +123,90 @@ This topic discusses how to import the Integration environment database. The dat
...
+   ```
 
 1. At the `MariaDB [main]>` prompt, enter `exit`.
 
 1. At the shell command prompt, enter the following command to re-create the database.
 
-    ```
-    zcat var/db.sql.tgz | sed -e 's/DEFINER[ ]*=[ ]*[^*]*\*/\*/' | mysql -h <db-host> -P <db-port> -p -u <db-user> <db-name>
-    ```
+   ```bash
```
Comment (hguthrie) [inline src/cloud/setup/first-time-setup-import-import.md]:
```suggestion
   ```shell
```

File: src/cloud/setup/first-time-setup-import-import.md [hunk 4]
```diff
@@ -124,88 +123,90 @@ This topic discusses how to import the Integration environment database. The dat
...
+   zcat var/db.sql.tgz | sed -e 's/DEFINER[ ]*=[ ]*[^*]*\*/\*/' | mysql -h <db-host> -P <db-port> -p -u   <db-user> <db-name>
+   ```
 
-    For example,
+   For example,
 
-    ```
-    zcat var/db.sql.tgz | sed -e 's/DEFINER[ ]*=[ ]*[^*]*\*/\*/' | mysql -h database.internal -p -u user main
-    ```
+   ```bash
```
Comment (hguthrie) [inline src/cloud/setup/first-time-setup-import-import.md]:
```suggestion
   ```shell
```

File: src/cloud/setup/first-time-setup-import-import.md [hunk 5]
```diff
@@ -124,88 +123,90 @@ This topic discusses how to import the Integration environment database. The dat
...
 
     {:.bs-callout-warning}
     The base URL _must_ end with a `/` character.
 
 1. Confirm the change by entering the following command:
 
-    ```
-    SELECT * from core_config_data;
-    ```
+   ```bash
```
Comment (hguthrie) [inline src/cloud/setup/first-time-setup-import-import.md]:
```suggestion
   ```shell
```

File: src/cloud/setup/first-time-setup-import-import.md [hunk 6]
```diff
@@ -124,88 +123,90 @@ This topic discusses how to import the Integration environment database. The dat
...
+   mysql -h <db-host> -P <db-port> -p -u <db-user> <db-name>
+   ```
 
 1. Show the contents of the `core_config_data` table.
 
-    ```
-    SELECT * from core_config_data;
-    ```
+   ```bash
```
Comment (hguthrie) [inline src/cloud/setup/first-time-setup-import-import.md]:
```suggestion
   ```shell
```

File: src/cloud/setup/first-time-setup-import-import.md [hunk 7]
```diff
@@ -124,88 +123,90 @@ This topic discusses how to import the Integration environment database. The dat
...
+   ```
 
     Note the `path` of `web/unsecure/base_url`; this is the value you'll change.
 
 1. Enter the following command to change the value of `path` to your integration server's unsecure base URL:
 
-    ```
-    UPDATE core_config_data SET value='<Cloud unsecure base URL>' WHERE path='web/unsecure/base_url';
-    ```
+   ```bash
```
Comment (hguthrie) [inline src/cloud/setup/first-time-setup-import-import.md]:
```suggestion
   ```shell
```

File: src/cloud/setup/first-time-setup-import-import.md [hunk 8]
```diff
@@ -219,52 +220,65 @@ The {{site.data.var.ee}} encryption key is required as an environment variable i
...
 
-    ```php
-    return array (
-      'crypt' =>
-      array (
-        'key' => '<your encryption key>',
-      ),
-    );
-    ```
+   ```php
```
Comment (hguthrie) [inline src/cloud/setup/first-time-setup-import-import.md]: This is a PHP excerpt. Per our style guide "When the PHP code block doesn't have the starting <?php tag, it won't be highlighted appropriately."

```suggestion
   ```php?start_inline=1
```

File: src/cloud/setup/first-time-setup-import-import.md [hunk 9]
```diff
@@ -219,52 +220,65 @@ The {{site.data.var.ee}} encryption key is required as an environment variable i
...
-    {:.bs-callout-info}
-    You must add this key to the `env.php` file for all environments: Integration, Staging, and Production.
+   {:.bs-callout-info}
+   You must add this key to the `env.php` file for all environments: Integration, Staging, and Production.
 
 ## Import media {#media}
 
 To import media files into your Cloud environment:
 
 1. If you haven't done so already, SSH to the Cloud environment.
```
Comment (hguthrie) [inline src/cloud/setup/first-time-setup-import-import.md]:
```suggestion
1. If you have not done so already, SSH to the Cloud environment.
```

File: src/guides/v2.2/release-notes/ReleaseNotes2.2.2EE.md [hunk 10]
```diff
@@ -691,7 +691,7 @@ You can find Magento Shipping-specific release notes in [Magento Shipping Releas
 
 **Issue**: In Swagger, the text area that contains the payload structure of some POST and PUT operations is not displayed. If a fraction of the text area is displayed, you can click on it to display the payload structure in a text area in the center of the page. If the text area is not displayed at all, then you cannot access the payload structure.
 
-**Workaround**: Use the [static Swagger site]({{ site.baseurl }}/swagger) to navigate to the REST call you want to use, then copy the payload structure to your Swagger instance.
+**Workaround**: Use the [static Swagger site]({{ site.baseurl }}/swagger/index_22.html) to navigate to the REST call you want to use, then copy the payload structure to your Swagger instance.
```
Comment (hguthrie) [inline src/guides/v2.2/release-notes/ReleaseNotes2.2.2EE.md]:
```suggestion
**Workaround**: Use the [static Swagger site]({{ site.baseurl }}/swagger/index_22.html) to navigate to the REST call, then copy the payload structure to your Swagger instance.
```

File: src/cloud/setup/first-time-setup-import-import.md [hunk 11]
```diff
@@ -219,52 +220,65 @@ The {{site.data.var.ee}} encryption key is required as an environment variable i
...
-To add your {{site.data.var.ee}} encryption key:
+{:.procedure}
+To add your {{site.data.var.ee}} encryption key using the `CRYPT_KEY` environment variable:
 
-1. If you haven't done so already, SSH to the Cloud environment.
+1. Log in to your Cloud project as an Admin user from the Project Web UI, or from the command line.
+
+1. Set the `CRYPT_KEY` environment variable to the encryption key value that you copied from the remote environment:
+
+   -  From the Project Web UI, select your environment, then select the *Variables* tab to set the `CRYPT_KEY` value. See [Set environment and project variables]({{site.baseurl}}/cloud/project/project-webint-basic.html#project-conf-env-var).
```
Comment (hguthrie) [inline src/cloud/setup/first-time-setup-import-import.md]:
```suggestion
   -  From the Project Web UI, select your environment, then select the _Variables_ tab to set the `CRYPT_KEY` value. See [Set environment and project variables]({{site.baseurl}}/cloud/project/project-webint-basic.html#project-conf-env-var).
```

File: src/cloud/setup/first-time-setup-import-import.md [hunk 12]
```diff
@@ -124,88 +123,90 @@ This topic discusses how to import the Integration environment database. The dat
...
 1. Show the contents of the `core_config_data` table.
 
-    ```
-    SELECT * from core_config_data;
-    ```
+   ```bash
+   SELECT * from core_config_data;
+   ```
 
     Note the `path` of `web/unsecure/base_url`; this is the value you'll change.
```
Comment (hguthrie) [inline src/cloud/setup/first-time-setup-import-import.md]:
```suggestion
    Note the `path` of `web/unsecure/base_url`; this is the value to change.
```

File: src/cloud/setup/first-time-setup-import-import.md [hunk 13]
```diff
@@ -124,88 +123,90 @@ This topic discusses how to import the Integration environment database. The dat
...
 ### Update base URLs {#baseurl}
 
 Before you can access Magento from your local Cloud development system, you must change the Base URLs in the Magento database. Base URLs are stored in the `core_config_data` table.
 
 The following example shows how to change _only_ the insecure URL but you can use the same procedure to change secure URLs as well.
 
+{:.procedure}
 To update the unsecure base URL:
 
 1. If you haven't already done so, SSH to the Cloud integration server.
```
Comment (hguthrie) [inline src/cloud/setup/first-time-setup-import-import.md]: There are a few of these intro sentences in this PR. Probably should check them for consistency.

```suggestion
1. If you have not already done so, use SSH to connect to the Cloud integration server.
```

File: src/cloud/setup/first-time-setup-import-import.md [hunk 14]
```diff
@@ -124,88 +123,90 @@ This topic discusses how to import the Integration environment database. The dat
 
 When importing data, you will need to drop and create a new database. If you have done any data you want to keep, [create a backup]({{ site.baseurl }}/cloud/project/project-webint-snap.html) of the database.
 
+{:.procedure}
 To drop and re-create the Cloud database:
 
 1. SSH to the Integration environment.
```
Comment (hguthrie) [inline src/cloud/setup/first-time-setup-import-import.md]:
```suggestion
1. Use SSH to log in to the Integration environment.
```

File: src/cloud/setup/first-time-setup-import-import.md [hunk 15]
```diff
@@ -124,88 +123,90 @@ This topic discusses how to import the Integration environment database. The dat
 
 When importing data, you will need to drop and create a new database. If you have done any data you want to keep, [create a backup]({{ site.baseurl }}/cloud/project/project-webint-snap.html) of the database.
```
Comment (hguthrie) [inline src/cloud/setup/first-time-setup-import-import.md]:
```suggestion
When importing data, you need to drop and create a new database. If you have data you want to keep, [create a backup]({{ site.baseurl }}/cloud/project/project-webint-snap.html) of the database.
```

File: src/cloud/docker/docker-database.md (hunk 1 above)
Comment (meker12) [inline src/cloud/docker/docker-database.md]:
```suggestion
When you import a database from an existing Magento installation into a new {{ site.data.var.ece }} environment, you must add the encryption key from the remote environment to the new environment, and then deploy the changes. See [Add the Magento encryption key]({{ site.baseurl}}/cloud/setup/first-time-setup-import-import.html#encryption-key).
```

**Pull request merged**